### ISSUE TITLE
Reproduce Paper Results

### ISSUE THREAD

**Issue by blainerothrock** [blainerothrock/earthquakes#3]
Using the current code base, reproduce the results of the [paper](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C14&q=Automating+the+Detection+of+Dynamically+triggered+Earthquakes+via+a+Deep+Metric+Learning+Algorithm&btnG=)

Data and installation instructions found in #1

**Comment by blainerothrock** [comment]
Running the initial testing script from the README resulted in bad results: 

This is a test of `SAC_20021102_XF` earthquake

```
Predict  0        1        
Actual
0        35       0        

1        42       15 
```

unsure why.

**Comment by blainerothrock** [comment]
Vivian shared a directory located on the GPGPU Machine which had a different version of that. This allowed for much better results that are closer the results reported in the paper.
```json
{
    "augmentations": "amplitude:noise",
    "batch_size": 64,
    "dataset_directory": "/home/b/dev/seis/vivian_earthquake/earthquakes/data/prepare_backup/trigger",
    "embedding_size": 10,
    "learning_rate": 2e-05,
    "length": 8192,
    "loss_function": "dpcl",
    "model_type": "conv",
    "num_epochs": 100,
    "num_workers": 10,
    "output_directory": "models/run12",
    "sample_strategy": "sequential",
    "split": "SAC_20021102_XF_prem",
    "training_strategy": "siamese",
    "transforms": "bandpass:whiten",
    "weight_decay": 0.1
}
```

```
Predict  0        1        
Actual
0        51       1        

1        16       59 
```

[combined.pdf](https://github.com/blainerothrock/earthquakes/files/4478418/combined.pdf)
